docs/install.rst

```diff
@@ -30,63 +29,119 @@ The easiest way to install Python and SciPy is with `Anaconda`_, a free scientif
 
 .. rubric:: Windows
 
-#. Go to the `Anaconda`_ website and download the Python 3.6 installer.
+#. Go to the `Anaconda`_ website and download the Python 3.8 (or newer) installer.
 
 #. Run it and accept the default options during the installation.
 
-#. Open `Anaconda Prompt` from the `Start` menu. Enter the following command a press enter. This will enable the `conda-forge`::
+#. Open `Anaconda Prompt` from the `Start` menu.
+
+#. Create a new environment for Pylake named pylake by invoking the following command::
+
+    conda create -n pylake
+
+#. The environment can then be activated::
+
+    conda activate pylake
+
+#. Verify that `(pylake)` is now prefixed to the path in your anaconda prompt (this means the environment is activated).
```

Comment:
If we're targeting non-expert users, we might want to phrase this more something like:
> The line where you enter your next command should now start with the text `(pylake)` instead of `(base)`, to indicate you're in the _pylake_ environment you just created.

Reply:
Good point! Amended!